gui.py

```diff
@@ -219,7 +219,7 @@
                     <p>Please contact <a href="mailto:uaf-snap-data-tools@alaska.edu">uaf-snap-data-tools@alaska.edu</a> if you have questions or would like to provide feedback for this tool. <a href="https://uaf-snap.org/tools-overview/">Visit the SNAP Climate + Weather Tools page</a> to see our full suite of interactive web tools.</p>
                     <p>Copyright © 2021 University of Alaska Fairbanks.  All rights reserved.</p>
                     <p>UA is an AA/EO employer and educational institution and prohibits illegal discrimination against any individual.  <a href="https://www.alaska.edu/nondiscrimination/">Statement of Nondiscrimination</a> and <a href="https://www.alaska.edu/records/records/compliance/gdpr/ua-privacy-statement/">Privacy Statement</a>.</p>
-                    <p>Photo © __________</p>
+                    <p>Photo © Anne Kokko</p>
```

Comment:
Before merging, let's confirm with Carolyn exactly what language should be used here!